src/components/OnBoarding/ThirdForm.jsx

```diff
@@ -0,0 +1,310 @@
+          <option key={ampm} value={ampm}>
+            {ampm}
+          </option>
+        ))}
+      </Select>
+    </Stack>
+  );
+
+  TimeInput.propTypes = {
+    label: propTypes.string.isRequired,
+    value: propTypes.shape({
+      hour: propTypes.string.isRequired,
+      minute: propTypes.string.isRequired,
+      ampm: propTypes.string.isRequired,
+    }).isRequired,
+    onChange: propTypes.func.isRequired,
+    isReadOnly: propTypes.bool.isRequired,
+  };
+
+  const handleTimeChange = (day, period, value) => {
+    changeBusinessHours({ ...businessHours, [day]: { ...businessHours[day], [period]: value } });
+  };
+
+  const [phoneNumber, changePhoneNumber] = useState('');
+  const [heardAbout, changeHeardAbout] = useState('');
+  const [termsAndConditionsIsOpened, changeTermsAndConditionsIsOpened] = useState(false);
+  const [termsAndConditionsAccepted, changeTermsAndConditionsAccepted] = useState(false);
+  const [submitAndAcceptedFalse, changeSubmitAndAcceptedFalse] = useState(false);
+  console.log(submitAndAcceptedFalse);
+  return (
+    <>
+      {termsAndConditionsIsOpened ? (
+        <Modal isOpen={true} size="3xl" isCentered>
+          <ModalOverlay>
+            <ModalContent alignItems="center">
+              <ModalHeader>
+                {' '}
+                <Text marginLeft="1vh" fontSize="2xl">
+                  Terms and Conditions{' '}
+                </Text>
+              </ModalHeader>
+              <ModalBody>
+                <Text fontWeight="bold" fontSize="sm">
+                  I hereby acknowledge and agree to serve as a member/volunteer for Feeding Pets of
+                  the Homeless, 710 West Washington Street, Carson City, NV 89703. I give my consent
+                  and agree to release, indemnify and hold harmless Feeding Pets of the Homeless
+                  (DBA: Pets of the Homeless) and all personnel, including but not limited to its
+                  officers, agents and/or employees, other participants, sponsors, advertisers, I
+                  hereby assume all of the risks of participating as a volunteer, with respect to
+                  any and all injury, disability, death or loss or damage of person or property,
+                  whether arising from negligence of the releases or otherwise, to the fullest
+                  extent permitted by law. I agree to comply with the guidelines and conditions for
+                  participation. See Fundraising Policies here:
+                  http://www.petsofthehomeless.org/fundraising-policy-for-pets-of-the-homeless/ Pets
+                  of the Homeless reserves the right to terminate this agreement at any time. I also
+                  grant Feeding Pets of the Homeless the right to use photographs of me at events
+                  and activities and use the photographs in future advertising including online
+                  webpage. I acknowledge that all information regarding Pets of the Homeless'
```

Comment:
🚫 **[eslint]** <[react/no-unescaped-entities](https://github.com/jsx-eslint/eslint-plugin-react/tree/master/docs/rules/no-unescaped-entities.md)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>`'` can be escaped with `&apos;`, `&lsquo;`, `&#39;`, `&rsquo;`.